src/helpers/getAllowlistMap.ts

```diff
@@ -0,0 +1,31 @@
+import * as fs from 'fs'
+import { resolve } from 'path'
+import AttestationType from '@/models/AttestationType'
+
+function getAllowlist(attestationType: AttestationType) {
+  try {
+    const filePath = resolve(
+      process.cwd(),
+      'merkleTrees',
+      `${attestationType}.txt`
```

Comment:
Sooner or later we need to remove `0, 1, 2...` into actual type names

Reply:
If we'll have too much types (like 20), we might mess things up

Reply:
Yes, but we think about it later